handlers/refunds_test.go

```diff
@@ -44,5 +44,31 @@ func TestUnitHandleCreateRefund(t *testing.T) {
 		w := httptest.NewRecorder()
 		HandleCreateRefund(w, req)
 		So(w.Code, ShouldEqual, http.StatusBadRequest)
+		print(w.Body)
```

Comment:
should the print statements be removed or intentional?

Reply:
Removed